Tiltfile

```diff
@@ -0,0 +1,79 @@
+# Parse config file
+config.define_bool("porch-server-debug")
+config.define_bool("porch-controllers-debug")
+config.define_bool("porch-function-runner-debug")
+cfg = config.parse()
+
+# There are cross-dependencies, so whenever a single file is changed, everything would be redeployed with TRIGGER_MODE_AUTO
+trigger_mode(TRIGGER_MODE_MANUAL)
+
+#TODO(nagygergo): Discover these automatically?
+crds = [
+    "api/porchconfig/v1alpha1/config.porch.kpt.dev_functions.yaml",
+    "api/porchconfig/v1alpha1/config.porch.kpt.dev_repositories.yaml",
+    "controllers/config/crd/bases/config.porch.kpt.dev_fleetmembershipbindings.yaml",
```

Comment:
The fleet stuff is junk left over from the migration from kpt and needs to be removed/moved to another repo. We don't use it so can be ignored. Same goes for the Function CRD above.